### PR TITLE
Added Bayesian One-Sample & Paired Samples Wilcoxon Tests

### PR DESCRIPTION
**Pull request by JohnnyDoorn**
I cannot request reviews apparently, but maybe Don?  (sorry Don! you rock! <3 <3)

### CODE REVIEW

**Comment by JorisGoosen** [comment]
Hey @JohnnyDoorn I just gave you maintainer permission I think, can you try to add @vandenman as reviewer now?

**Comment by vandenman** [comment]
@JohnnyDoorn if you type @vandenman then at least I get a notification :)

**Comment by JorisGoosen** [comment]
Yeah, my previous comment was an attempt to tell @JohnnyDoorn subtly that he forgot that :p

**Comment by JohnnyDoorn** [comment]
Hahaha yeah I know that, but it was more of a joke...

File: R/ttestbayesianpairedsamples.R [hunk 1]
```diff
@@ -153,12 +224,114 @@ TTestBayesianPairedSamples <- function(jaspResults, dataset, options) {
...
+.sampleGibbsOneSampleWilcoxon <- function(diffScores, nIter = 10, rscale = 1/sqrt(2)){
+  ybar <- mean(diffScores)
+  n <- length(diffScores)
+  sigmaSq <- 1
+  mu <- ybar
+  g <- ybar^2 / sigmaSq + 1
+  
+  for(i in 1:nIter){   
+    #sample mu
+    varMu  = sigmaSq / (n + (1 / g))
```
Comment (vandenman) [inline R/ttestbayesianpairedsamples.R:326]: `<-` rather than `=`

**Comment by JorisGoosen** [comment]
@JohnnyDoorn can you fix this? Aka, rebase and push and all that?

And is this code in jasp-desktop or is this basically floating around?

**Comment by vandenman** [comment]
@JorisGoosen the code is already in jasp-desktop (https://github.com/jasp-stats/jasp-desktop/pull/4037)

**Comment by JorisGoosen** [comment]
Ok then we dont need this anymore as I will be copying everything here soon...

**Pull request closed without merging**